docs/SPIR-V.rst

```diff
@@ -1701,15 +1701,21 @@ The SPIR-V ``InvocationID`` (``SV_OutputControlPointID`` in HLSL) is used to ind
 into the InputPatch and OutputPatch arrays to read/write information for the given
 vertex.
 
+The hull main entry function in HLSL returns only one value (say, of type ``T``), but 
+that function is in fact executed several times. Therefore, we create a stage output variable
+that is an array with elements of type ``T``. The number of elements of the array is equal to
+the number of output control points.
```

Comment:
the number of output control points. Each final output control point is written into the corresponding element in the array using SV_OutputControlPointID as the index.